pulp_rpm/tests/functional/api/test_consume_content.py

```diff
@@ -39,6 +39,9 @@ def setUpClass(cls):
         cls.pkg_mgr = cli.PackageManager(cls.cfg)
         cls.pkg_mgr.raise_if_unsupported(unittest.SkipTest, "This test requires dnf or yum.")
 
+    def _has_dnf(self):
+        return self.pkg_mgr.name == "dnf"
```

Comment:
Instead of doing this, would it be possible to use the pulp-smash functionality? https://github.com/pulp/pulp-smash/blob/master/pulp_smash/cli.py#L715

Reply:
Or extend it

Reply:
> Instead of doing this, would it be possible to use the pulp-smash functionality? https://github.com/pulp/pulp-smash/blob/master/pulp_smash/cli.py#L715

As written the tests rely on functionality that dnf has, and yum does not. See https://github.com/pulp/pulp_rpm/blob/f29eab685fb2fd44129bb453033661670f5e86b9/pulp_rpm/tests/functional/api/test_consume_content.py#L91 , for example. 

This test needs serious rewriting - see https://pulp.plan.io/issues/7891#note-7. The change here is just to make it possible to run on CentOS7, the false failures make it hard to trust test runs.